packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -40,13 +49,272 @@ type GetResolverParams = {
+    )
+    const customResolver = customResolvers[title][path][method] as ResolveObject
+    if (typeof customResolver.subscribe === 'function') {
+      return customResolver.subscribe
+    }
+  }
+
+  return (root: any, args, ctx, info = {}) => {
+    /**
+     * Determine possible topic(s) by resolving callback path
+     *
+     * GraphQL produces sanitized payload names, so we have to sanitize before
+     * lookup here
+     */
+
+    const paramName = Oas3Tools.sanitize(
+      payloadName,
+      Oas3Tools.CaseStyle.camelCase
+    )
+
+    let resolveData: any = {}
+
+    if (
+      root &&
+      typeof root === 'object' &&
+      typeof root['_openAPIToGraphQL'] === 'object' &&
+      typeof root['_openAPIToGraphQL'].data === 'object'
+    ) {
+      const parentIdentifier = getParentIdentifier(info)
+      if (
+        !(parentIdentifier.length === 0) &&
+        parentIdentifier in root['_openAPIToGraphQL'].data
+      ) {
+        /**
+         * Resolving link params may change the usedParams, but these changes
+         * should not be present in the parent _openAPIToGraphQL, therefore copy
+         * the object
+         */
+        resolveData = JSON.parse(
+          JSON.stringify(root['_openAPIToGraphQL'].data[parentIdentifier])
+        )
+      }
+    }
+
+    if (payloadName && typeof payloadName === 'string') {
+      // The option genericPayloadArgName will change the payload name to "requestBody"
+      const sanePayloadName = data.options.genericPayloadArgName
+        ? 'requestBody'
+        : Oas3Tools.sanitize(payloadName, Oas3Tools.CaseStyle.camelCase)
+
+      if (sanePayloadName in args) {
+        if (typeof args[sanePayloadName] === 'object') {
+          const rawPayload = Oas3Tools.desanitizeObjKeys(
+            args[sanePayloadName],
+            data.saneMap
+          )
+          resolveData.usedPayload = rawPayload
+        } else {
+          const rawPayload = JSON.parse(args[sanePayloadName])
+          resolveData.usedPayload = rawPayload
+        }
+      }
+    }
+
+    pubsubLog(
+      `Subscription schema : ${JSON.stringify(resolveData.usedPayload)}`
+    )
+
+    if (typeof resolveData.usedParams === 'undefined') {
+      resolveData.usedParams = {}
+    }
+
+    if (connectOptions) {
+      resolveData.usedRequestOptions = connectOptions
+    } else {
+      resolveData.usedRequestOptions = {
+        method: resolveData.usedPayload.method
+          ? resolveData.usedPayload.method
+          : method.toUpperCase()
+      }
+    }
+
+    let value = path
+    let paramNameWithoutLocation = paramName
+    if (paramName.indexOf('.') !== -1) {
+      paramNameWithoutLocation = paramName.split('.')[1]
+    }
+
+    // /**
+    //  * see if the callback path contains constants expression
+    //  *
+    //  */
+    if (value.search(/{|}/) === -1) {
+      args[paramNameWithoutLocation] = isRuntimeExpression(value)
+        ? resolveLinkParameter(paramName, value, resolveData, root, args)
```

Comment:
Sure point, that's probably why you got misled in your previous comment.
But since the callback path is a runtime expression, like the link parameter ...